### ISSUE TITLE
Stuck at "Less than a minute remaining"

### ISSUE THREAD

**Issue by JWIMaster** [tylernguyen/x1c6-hackintosh#205]
I have followed the guide, and the steps to install macos on the factory drive, however i am stuck on "less than a minute remaining". My config.plist has been edited to add the "ssdt.nvme.aml" and the hackernvme kext, other than that (and wifi kext) no other edits have been made. I have tried multiple installations.

**Comment by JWIMaster** [comment]
![IMG_0422](https://github.com/tylernguyen/x1c6-hackintosh/assets/64947174/69dea786-8927-4c70-a918-cf1423e0ba50)

**Comment by Dwigth** [comment]
Did you solve it?

**Comment by tigazq** [comment]
did you solve it?

**Comment by JWIMaster** [comment]
Yes. It's because of the NVMe drive. Don’t try using the PM981, it's more work than it’s worth.
